Makefile

```diff
@@ -1,7 +1,10 @@
-.PHONY: build test clean
-build:
+.PHONY: build generate test clean
+build:generate
 	go build -o output/pb-go main/main.go
```

Comment:
Seperate for **Client** `main.go` and **Server** `main.go`. Client is in construction.